pos_stock_lot_required/static/src/js/screens.js

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2019 Druidoo - Iván Todorovich
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+   
+odoo.define('pos_stock_lot_required.screens', function (require) {
+"use strict";
+
+var screens = require('point_of_sale.screens');
+
+screens.ActionpadWidget.include({
+    renderElement: function() {
+        var self = this;
+        this._super();
+        this.$('.pay').click(function(){
+            var order = self.pos.get_order();
+            var has_valid_product_lot = _.every(order.orderlines.models, function(line){
+                return line.has_valid_product_lot();
+            });
+            if(!has_valid_product_lot){
+                self.gui.show_popup('alert', {
+                    title: _t('Empty Serial/Lot Number'),
+                    'body':  _t('One or more product(s) required serial/lot number.'),
+                });
+            }else{
+                self.gui.show_screen('payment');
+            }
+        });
+        this.$('.set-customer').click(function(){
+            self.gui.show_screen('clientlist');
+        });
```

Comment:
```suggestion
```

This is not necessary because it's already handled by `this._super()`.
By adding this here you're calling `show_screen('clientlist');` twice, which has no functional impact but it's not clean